cli/src/commands/util/exec.rs

```diff
@@ -66,27 +66,55 @@ use crate::ui::Ui;
 pub(crate) struct UtilExecArgs {
     /// External command to execute
     command: String,
+
     /// Arguments to pass to the external command
     #[arg(value_hint = clap::ValueHint::FilePath)]
     args: Vec<String>,
+
+    // If set, command is treated as a path relative to the workspace root.
+    #[arg(long)]
+    relative_command: bool,
```

Comment:
Perhaps `ignore_environment_path` or `ignore_path_dirs` with a short option, e.g. `-P`?

Regardless, you could add "The directories listed in the `PATH` environment variable are not searched" to the help.

Reply:
I'll resolve in a follow-up PR and do transparent first